CONTRIBUTING.md

```diff
@@ -59,9 +59,7 @@ Here's how to make a one-off code change. (If you're working on a larger feature
   * Before making a commit, start up a local instance of Oppia and do some manual testing in order to check that you haven't broken anything! Also, ensure that all automated tests still pass:
 
     ```
-      bash scripts/test.sh
-      bash scripts/run_js_tests.sh
-      bash scripts/run_integration_tests.sh (if necessary)
+      bash scripts/run_tests.sh
```

Comment:
Per our discussion, I'm wondering whether we should encourage contributors to only use run_tests.sh before mentioning it is ready to merge into develop/the target branch, but otherwise to focus on running run_{backend,frontend}_tests.sh during regular development. This will encourage them to save time and not wait on the protractor tests until necessary.